testing/libkml/APKBUILD

```diff
@@ -13,14 +13,21 @@ makedepends="
 	cmake
 	curl-dev
 	expat-dev
-	gtest-dev
 	minizip-dev
 	openjdk$_jdkver-jdk
 	python3-dev
 	swig
 	uriparser-dev
 	zlib-dev
 	"
+checkdepends="
+	gtest-dev
+	"
+depends_dev="
```

Comment:
isn't it better to declare depends_dev before makedepends and then add $depends_dev to makedepends ?

Reply:
Yes, that's much better. I changed that and force pushed.